src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/PublicAPI.Shipped.txt

```diff
@@ -5,7 +5,13 @@ NuGet.VisualStudio.Internal.Contracts.AlternatePackageMetadataContextInfo.Packag
 NuGet.VisualStudio.Internal.Contracts.AlternatePackageMetadataContextInfo.VersionRange.get -> NuGet.Versioning.VersionRange!
 NuGet.VisualStudio.Internal.Contracts.IInstalledAndTransitivePackages
 NuGet.VisualStudio.Internal.Contracts.IInstalledAndTransitivePackages.InstalledPackages.get -> System.Collections.Generic.IReadOnlyCollection<NuGet.VisualStudio.Internal.Contracts.IPackageReferenceContextInfo!>!
-NuGet.VisualStudio.Internal.Contracts.IInstalledAndTransitivePackages.TransitivePackages.get -> System.Collections.Generic.IReadOnlyCollection<NuGet.VisualStudio.Internal.Contracts.IPackageReferenceContextInfo!>!
+NuGet.VisualStudio.Internal.Contracts.ImplicitProjectAction
```

Comment:
please check this file. It's very difficult for us to determine whether your branch has changed the order of entries in this file, or if you're introduced breaking API changes, or incorrectly added new APIs to this file, rather than the unshipped.txt.  I suggest just reverting this file back to the dev branch contents, then the analyzer will tell you if something is wrong.

Reply:
Fixed formatting.